datapackage.json

```diff
@@ -0,0 +1,217 @@
+{
+  "name": "Unicode-Codes",
+  "title": "Unicode-Codes",
+  "resources": [
+    {
+      "path": "archive/source.csv",
+      "pathType": "local",
```

Comment:
@solo11 not sure which tool did you use to generate this. I think this one looks outdated.